ui/component/app/view.jsx

```diff
@@ -174,6 +177,52 @@ function App(props: Props) {
     return () => window.removeEventListener('keydown', handleKeyPress);
   }, []);
 
+  // Enable ctrl +/- zooming on Desktop.
+  // @if TARGET='app'
+  useEffect(() => {
```

Comment:
Could you move this to `effects/use-zoom.js`?

Trying to avoid this file getting massive. Then you can just do
```
// @if TARGET='app'
useZoom();
// @endif
```